lib/ansible/modules/windows/win_copy.py

```diff
@@ -44,59 +33,89 @@
     - Remote absolute path where the file should be copied to. If src is a
       directory, this must be a directory too.
```

Comment:
@sm4rk0 Exactly, **win_copy** will not create a directory, that's what **win_file** is for.